homeassistant/components/nexia/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "nexia",
+  "name": "Nexia",
+  "requirements": [
+    "nexia==0.4.1"
+  ],
+  "dependencies": [],
+  "codeowners": [
+    "@ryannazaretian", "@bdraco"
```

Comment:
Is ryannazaretian involved?

Reply:
Yes. He worked up the latest version of the PR, I did all the cleanup.  He is quite busy with work at the moment though so its falling to me to get it over the finish line.